frontend/.gitignore

```diff
@@ -1,8 +0,0 @@
-# files/directories to be ignored
-
-.sass-cache/
-
-node_modules/
-bower_components/
-
-dist/
```

Comment:
@aka-jain : See the .gitignore of root directory. 
there is `dist/` (line 14) that's working for the `frontend/dist/` .
However run project in locally and see this if everything is fine.